packages/connect-thegraph-voting/src/queries/index.ts

```diff
@@ -10,7 +10,6 @@ export const ALL_VOTES = (type: string) => gql`
       orgAddress
       creator
       metadata
-      open
```

Comment:
I am not sure if we should remove this? This is the error I get when leaving it:

> CombinedError: [GraphQL] Type `Vote` has no field `open`

Reply:
Yes, we should. The latest version of the schema doesn't have that property.